src/server/utilities/cspHeader/index.js

```diff
@@ -97,6 +97,9 @@ const directives = {
       'https://*.akamaihd.net',
       'https://cdn.optimizely.com/',
       'https://logx.optimizely.com/',
+      'https://europe-west1-bbc-otg-traf-mgr-bq-prod-4591.cloudfunctions.net', // Web-Vitals monitoring
```

Comment:
Does this still need to be used after moving to `https://ws.bbc-reporting-api.app`?

Reply:
Nope, good catch - looks like a merge mistake